Helper/Paymentorder.php

```diff
@@ -185,10 +185,6 @@ public function createPaymentorderObject($consumerProfileRef = null)
          * $paymentOrderItems = $this->createItemsObject();
          */
 
-        if ($consumerProfileRef) {
```

Comment:
With this change, `consumerProfileRef` is never submitted? Doesn't that mean Checkin is made non-functional in practice?

Reply:
Since we are not sending `payer` object anymore, we cannot send `consumerProfileRef`

Reply:
Without sending `consumerProfileRef`, Checkin is made non-functional, no?

Reply:
The `consumerProfileRef` is a unique value for each order. And even if we don't send it, it is automatically generated from SwedbankPay API, right?

Reply:
If `consumerProfileRef` is not submitted, no Checkin will be performed, the Payment Menu will have no idea who the payer is and will not display stored credit cards and other payment details.

Reply:
We have updated the functionality so that `consumerProfileRef` is sent with the `payer` object and it is unique per `paymentorder` request